src/Builder.php

```diff
@@ -114,6 +116,12 @@ public function setBarcodeType($type)
         throw new \LogicException('Unsupported barcode format');
     }
 
+    public function setBarcodeLabel($barcodeLabel)

```

Comment:
This is not necessary from the builder perspective. Builder knows nothing about label.

Reply:
I think same but is not crazy when you use Builder main class for setParams.

Reply:
It is worth to diffientate what is real param here. Single printer may be used to produce many different barcodes. So in such perspective barcodeLabel is not Printer parameter at all. It belongs to single label scale (the outpu() method)